tests/InterNations/Tests/Sniffs/Syntax/Fixtures/MethodTypeHints/MissingParameters.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+class MissingParameters
+
+{
+    /**
+     * @Route(
+     *  "/{context}/{entityType}/{entityId}",
+     *  name="comment_comment_post",
+     *  requirements={"entityId" = "\d+", "entityType" = "\w+", "context" = "\w+"},
+     *  defaults={"_format"="json"}
+     * )
+     * @Secure(roles="ROLE_USER")
+     * @Template("@InterNationsLayout/Formats/jquery.json.twig")
+     * @Method("POST")
+     * @return String[]
+     */
+    public function postAction(Request $request, string $context, string $entityType, int $entityId): array
+    {
+        $form = $this->createForm(CommentFormType::class);
```

Comment:
Can you remove everything non-essential, like the body and most of the doc blocks?